lib/order.rb

```diff
@@ -2,23 +2,42 @@
 require_relative './invoice.rb'
 
 class Order
```

Comment:
One thing I might say is I find this class name here a little confusing because you also have the instance variable 'order', you would end up called order.order - so maybe some re-naming here might make it slightly clearer to a new user